PWGHF/D2H/Tasks/taskLc.cxx

```diff
@@ -37,10 +38,36 @@ struct HfTaskLc {
   Configurable<double> yCandGenMax{"yCandGenMax", 0.5, "max. gen particle rapidity"};
   Configurable<double> yCandRecoMax{"yCandRecoMax", 0.8, "max. cand. rapidity"};
   Configurable<std::vector<double>> binsPt{"binsPt", std::vector<double>{hf_cuts_lc_to_p_k_pi::vecBinsPt}, "pT bin limits"};
+  // ThnSparse for ML outputScores and Vars
+  Configurable<bool> applyMl{"applyMl", false, "Flag to apply ML selections"};
+  ConfigurableAxis thnConfigAxisPt{"thnConfigAxisPt", {360, 0, 36}, ""};
+  ConfigurableAxis thnConfigAxisMass{"thnConfigAxisMass", {600, 1.98, 2.58}, ""};
+  ConfigurableAxis thnConfigAxisPtProng0{"thnConfigAxisPtProng0", {500, 0, 50}, ""};
+  ConfigurableAxis thnConfigAxisPtProng1{"thnConfigAxisPtProng1", {500, 0, 50}, ""};
+  ConfigurableAxis thnConfigAxisPtProng2{"thnConfigAxisPtProng2", {500, 0, 50}, ""};
```

Comment:
Coudn't this be the same configurable? Do you need different binning for the different daughter tracks?

Reply:
yes, I use one configurable in the new commit